packages/core/src/Modules/Cashier/Components/payment-agent-details.jsx

```diff
@@ -3,24 +3,20 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import { Icon } from '@deriv/components';
 
-class Detail extends React.Component {
-    render() {
-        const { action, icon, is_last_child, value, ...rest } = this.props;
-
-        return (
-            <div className='payment-agent__accordion-content-line'>
-                <Icon icon={`Ic${icon}`} className='payment-agent__accordion-content-icon' color='secondary' />
-                <a
-                    className='payment-agent__contact cashier__paragraph'
-                    href={`${action ? `${action}:` : ''}${value}`}
-                    {...rest}
-                >
-                    {value}
-                </a>
-            </div>
-        );
-    }
-}
+const Detail = ({ action, icon, is_last_child, value, ...rest }) => {
+    return (
+        <div className='payment-agent__accordion-content-line'>
+            <Icon icon={`Ic${icon}`} className='payment-agent__accordion-content-icon' color='secondary' />
+            <a
+                className='payment-agent__contact cashier__paragraph'
+                href={`${action ? `${action}:` : ''}${value}`}
+                {...rest}
+            >
+                {value}
```

Comment:
(suggestion) for a more intuitive API, can we use `children` instead of value for this?